lms/startup.py

```diff
@@ -184,6 +184,10 @@ def course_display_name_sub(user, course):
         """ Returns the course's display name """
         return course.display_name
 
+    def course_id_sub(user, course):
```

Comment:
Can we give this a better name? The current function name doesn't make its intent clear.

Also, the docstring text doesn't match the function logic.

Can you clarify?